pilot/pkg/networking/core/v1alpha3/listener_test.go

```diff
@@ -113,6 +136,98 @@ var (
 	}
 )
 
+func TestInboundListenerConfigProxyV13(t *testing.T) {
+	_ = os.Setenv(features.EnableProtocolSniffing.Name, "true")
+	defer func() { _ = os.Unsetenv(features.EnableProtocolSniffing.Name) }()
+
+	for _, p := range []*model.Proxy{&proxy13, &proxy13HTTP10} {
+		testInboundListenerConfigV13(t, p,
+			buildService("test1.com", wildcardIP, protocol.HTTP, tnow.Add(1*time.Second)),
+			buildService("test2.com", wildcardIP, "unknown", tnow),
+			buildService("test3.com", wildcardIP, protocol.HTTP, tnow.Add(2*time.Second)))
+		testInboundListenerConfigV13(t, p,
```

Comment:
this is redundant isn't it?

Reply:
I will remove it since inbound listener doesn't have conflict.